app/controllers/events_controller.rb

```diff
@@ -20,29 +21,35 @@ class EventsController < ApplicationController
     end
   end
 
+  rescue_from GoogleCalendar::EventCreator::EventInvalidRoom do |error|
+    render json: error.message, status: :unprocessable_entity
+  end
+
   rescue_from Google::Apis::AuthorizationError do
     session.delete(:credentials)
     render json: {error: 'Authorization error'}, status: :unauthorized
   end
 
-  rescue_from GoogleEvent::EventInTimeSpanError do |message|
+  rescue_from GoogleCalendar::EventCreator::EventInTimeSpanError do |message|
     render json: {conference_room_id: [message]}, status: :unprocessable_entity
   end
 
   def index
-    render json: GoogleEventLister.new(session[:credentials], session[:email]).call(TimeInterval.week(date_param))
+    time_interval = TimeInterval.week(date_param)
+    events = google_event_client.list_events(time_interval.start.to_datetime, time_interval.end.to_datetime)
```

Comment:
I would pass time_interval instead of start_time, end_time
`events = google_event_client.list_events(time_interval)
`